namd/src/colvarproxy_namd.h

```diff
@@ -46,6 +46,9 @@ class colvarproxy_namd : public colvarproxy, public GlobalMaster {
   /// Pointer to the NAMD simulation input object
   SimParameters *simparams;
 
+  /// Pointer to Controller object
+  Controller const *controller;
```

Comment:
Nitpick: I think we should start using `unique_ptr` for these members. Doesn't have to be now of course, but do you see any issue with this in the future?

Reply:
Agreed! I think that should be its own update where we do the change in bulk.